Firestore/Example/Tests/Local/FSTPersistenceTestHelpers.mm

```diff
@@ -47,10 +49,14 @@ + (NSString *)levelDBDir {
 }
 
 + (FSTLevelDB *)levelDBPersistence {
+  // This ownes the database-ids since we do not have firestore-client instance to own them.
```

Comment:
Typo: s/ownes/owns/

Nit: it's useful to refer to classes (like DatabaseId) by their class names so that refactor searches turn them up. I'd suggest changing "database-ids" to "DatabaseIds" and firestore-client to FirestoreClient.

Reply:
done